src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -709,6 +709,13 @@
       }
     }
 
+    //fixed with this mixin because the IE11 and IE10 mixins didnt work
+    @include fixForIE9AndUp() {
+      .pagination-current {
+        line-height: 0.5rem;
```

Comment:
lets use line-height: normal here instead. That will fix the issue in IE11.

Reply:
changed. works fine.